### PR TITLE
Fix the float/double mixup on the swift side

### PR DESCRIPTION
**Pull request by luanpotter**

### CODE REVIEW

File: darwin/Classes/NotificationsHandler.swift [hunk 1]
```diff
@@ -161,7 +161,7 @@ class NotificationsHandler {
             MPMediaItemPropertyArtist: artist,
             MPMediaItemPropertyPlaybackDuration: duration,
             MPNowPlayingInfoPropertyElapsedPlaybackTime: elapsedTime,
-            MPNowPlayingInfoPropertyPlaybackRate: playbackRate
+            MPNowPlayingInfoPropertyPlaybackRate: Float(playbackRate)
```
Comment (erickzanardo) [inline darwin/Classes/NotificationsHandler.swift:164]: Shouldn't here be double as well?

File: darwin/Classes/WrappedMediaPlayer.swift [hunk 2]
```diff
@@ -103,14 +103,14 @@ class WrappedMediaPlayer {
...
         self.volume = volume
-        player?.volume = volume
+        player?.volume = Float(volume)
     }
     
-    func setPlaybackRate(playbackRate: Float) {
+    func setPlaybackRate(playbackRate: Double) {
         self.playbackRate = playbackRate
-        player?.rate = playbackRate
+        player?.rate = Float(playbackRate)
```
Comment (erickzanardo) [inline darwin/Classes/WrappedMediaPlayer.swift:113]: And here

File: darwin/Classes/WrappedMediaPlayer.swift [hunk 3]
```diff
@@ -298,7 +298,7 @@ class WrappedMediaPlayer {
             duckAudio: duckAudio
         ) {
             player in
-            player.volume = volume
+            player.volume = Float(volume)
```
Comment (erickzanardo) [inline darwin/Classes/WrappedMediaPlayer.swift:301]: And here

**Review by erickzanardo — APPROVED** [review]
Some questions but LGTM

File: darwin/Classes/NotificationsHandler.swift (hunk 1 above)
Comment (luanpotter) [reply darwin/Classes/NotificationsHandler.swift:164]: can't change the ios APIs :p

File: darwin/Classes/WrappedMediaPlayer.swift (hunk 2 above)
Comment (luanpotter) [reply darwin/Classes/WrappedMediaPlayer.swift:113]: can't change the ios APIs :p

File: darwin/Classes/WrappedMediaPlayer.swift (hunk 3 above)
Comment (luanpotter) [reply darwin/Classes/WrappedMediaPlayer.swift:301]: can't change the ios APIs :p

**Pull request merged**